cli/cmd/service_test.go

```diff
@@ -706,6 +715,97 @@ func ExampleServicedCLI_CmdServiceEdit_err() {
 	// service not found
 }
 
+func ExampleServicedCLI_CmdServiceConfigList_usage() {
+  InitServiceAPITest("serviced", "service", "config", "list")
+	// Output:
```

Comment:
tab inconsistency lines [720..732].

Actually, there are other tabbing issues on other lines below

Reply:
This is what I get for switching away from vim. Addressing.